morgoth/base/middleware.py

```diff
@@ -0,0 +1,9 @@
+from time import sleep
+
+from django.conf import settings
+
+
+class SleepMiddleware(object):
+    def process_request(self, request):
+        if settings.DEBUG:
+            sleep(getattr(settings, 'SLEEP_SECONDS', 0))
```

Comment:
Instead of keying this off of `DEBUG`, in Normandy we would have made a new setting enabling or disabling sleep, and make it part of the development profile. I think this makes it clearer what a setting does, instead of `DEBUG` which has a million effects. You could even make this `if settings.SLEEP_SECONDS > 0`, to make that setting the trigger.